src/dbt_client/dbtCoreIntegration.ts

```diff
@@ -223,17 +215,19 @@ export class DBTCoreProjectIntegration
 
   private async createPythonDbtProject() {
     await this.python.ex`from dbt_integration import *`;
+    const profilesDir = await this.findProfilesDirectory();
+    const targetPath = setEnvVariableValue("DBT_TARGET_PATH");
```

Comment:
What's this line? This should get call the target_path() you created in the Python bridge.